service/models.py

```diff
@@ -44,14 +51,13 @@ class Status(enum.Enum):
     updated_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow, onupdate=datetime.utcnow)
 
     def __repr__(self):
-        return "<Inventory %r id=[%s]>" % (self.name, self.id)
+        return "<Inventory %r id=[%s]>" % (self.name, self.product_id)
```

Comment:
Can add `self.condition.value` here since it is useful to identify a record.

Reply:
yeah can add that in as well

Reply:
Done